### DISCUSSION TITLE
Enchentes RS | Recrutamento de devs VOLUNTÁRIOS para desenvolvimento de uma plataforma para logística de doações e voluntariado

### DISCUSSION THREAD

**Discussion by thiagoblauthferreira** [community/community#123828] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Olá comunidade, sou de Porto Alegre e estou vivendo de perto o caos que nossa cidade e estado está passando com a enchentes! Eu graças estou bem mais familiares e amigos tiveram casas inundadas! Estou acompanhando de perto diversos grupos de voluntários e mapeei as principais dificuldades que os grupos estão enfrentando no processo de doações. Consegui pensar e organizar um fluxo ideal de informações que pode minimizar muito esses problemas, fiz até telas básicas de um protótipo mas meus conhecimentos vão até ai, estou na cara e coragem recrutando devs pra essa iniciativa. A plataforma pensada não parece complexa. 

Me desculpem se esse espaço não é o correta para esse tópico! Qualquer ajuda no projeto será super bem vinda.

Valeu :)

**Comment by CiceroLino** [comment]
O projeto já existe, https://github.com/SOS-RS/

**Comment by ebndev** [comment]
Hi @thiagoblauthferreira, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
